### PR TITLE
Feature: Absorb Victory Native

### PR DESCRIPTION
**Pull request by gksander**
This is the big one. Absorbs Victory Native into this repo to streamline native development (which will allow us to sunset the existing `victory-native` repo). This is the feature branch that other smaller PRs have been made against. To see some of the smaller units of work that have gone into this, see some PRs:

- [The initial prototype](https://github.com/FormidableLabs/victory/pull/1974)
- [Linting some of the files that moved over](https://github.com/FormidableLabs/victory/pull/1991)
- [Adding some tests, adding that to CI](https://github.com/FormidableLabs/victory/pull/2012)
- [Moving over some docs](https://github.com/FormidableLabs/victory/pull/2013)
- [Updating issue template, because users will now submit VN issues here](https://github.com/FormidableLabs/victory/pull/2014)

### CODE REVIEW

File: demo/rn/.expo-shared/assets.json [hunk 1]
```diff
@@ -0,0 +1,4 @@
+{
+  "12bb71342c6255bbf50437ec8f4441c083f47cdb74bd89160c15e4f43e52a1cb": true,
```
Comment (ryan-roemer) [inline demo/rn/.expo-shared/assets.json:2]: Is this supposed to be in git source?

File: .github/workflows/ci.yml [hunk 2]
```diff
@@ -13,6 +13,7 @@ on:
   pull_request:
     branches:
       - main
+      - feature/absorb-native-repo # TODO: REMOVE when feature branch has been merged to main
```
[the remark this answers is not in the document]
Comment (gksander) [reply .github/workflows/ci.yml]: Good catch!

File: demo/rn/src/screens/components-screen.tsx [hunk 3]
```diff
@@ -0,0 +1,97 @@
...
+  //     { key: "CreateContainerView", title: "Custom Containers" }
+  //   ],
+  //   title: "Containers"
+  // },
+  {
+    data: [
+      { key: "Legends", title: "Legends" },
+      { key: "Axis", title: "Axis" },
+      { key: "PolarAxis", title: "VictoryPolarAxis" }
+      // { key: "ErrorsTooltipsView", title: "Errors & Tooltips" }
```
Comment (ryan-roemer) [inline demo/rn/src/screens/components-screen.tsx:93]: Remove?

File: packages/victory-native/src/components/victory-brush-container.js [hunk 4]
```diff
@@ -0,0 +1,67 @@
...
+RectWithStyle.propTypes = {
+  style: PropTypes.object
+};
+
+const nativeBrushMixin = (base) =>
+  class VictoryNativeSelectionContainer extends base {
+    // eslint-disable-line max-len
+    // assign native specific defaultProps over web `VictoryBrushContainer` defaultProps
+    static defaultProps = {
+      ...VictoryBrushContainer.defaultProps,
```
Comment (ryan-roemer) [inline packages/victory-native/src/components/victory-brush-container.js:27]: MINOR: Use `Object.assign()` pattern for consistency with other components?

File: demo/rn/src/screens/components-screen.tsx [hunk 5]
```diff
@@ -0,0 +1,97 @@
...
+      { key: "Bar", title: "VictoryBar" },
+      { key: "Histogram", title: "VictoryHistogram" },
+      { key: "Scatter", title: "VictoryScatter" },
+      { key: "BoxPlot", title: "VictoryBoxPlot" },
+      { key: "ErrorBar", title: "VictoryErrorBar" },
+      { key: "Voronoi", title: "VictoryVoronoi" }
+    ],
+    title: "Charts"
+  },
+  // {
```
[the remark this answers is not in the document]
Comment (gksander) [reply demo/rn/src/screens/components-screen.tsx:81]: This is a good question. I pretty much just ported the existing demo over with as little tweaking as possible, which meant leaving in some of these comments. I'm happy to strip these out! But I'm not sure if somebody was leaving these around intentionally as something of a TODO.

What are your thoughts? Strip 'em?

**Review by ryan-roemer — COMMENTED** [review]
The infrastructure and build stuff looks great! Can't comment on substantive stuff 😛 

Great work!

**Review by boygirl — APPROVED** [review]
Thank you so much for this work! I think this could be published as a semver minor, but let me know what you think. I'll handle the `lerna publish` and let you know if I hit any fun new snags. I'll also take care of deprecating the old `victroy-native` repo.

**Comment by gksander** [comment]
I think a semver minor makes sense! Going to merge now, and 🤞 no major hiccups when running the publish!

**Pull request merged**